index.html

```diff
@@ -737,8 +737,8 @@ <h1>Term Codec Registry</h1>
         </tr>
         <tr>
           <td><code>0x32</code></td>
-          <td></td>
-          <td>Reserved for future use.</td>
+          <td>https://purl.imsglobal.org/spec/ob/v3p0/context.json</td>
```

Comment:
```suggestion
          <td>https://purl.imsglobal.org/spec/ob/v3p0/context-3.0.3.json</td>
```
I highly recommend we point to the latest.